REQUIRE

```diff
@@ -1,3 +1,3 @@
 julia 0.4
-DataStructures 0.3.8
+DataStructures 0.5.3
```

Comment:
this should be 0.5.0 https://github.com/JuliaCollections/DataStructures.jl/compare/v0.4.6...v0.5.0